src/engine/source/conf/include/conf/keys.hpp

```diff
@@ -21,6 +21,7 @@ constexpr std::string_view INDEXER_SSL_CERTIFICATE = "/indexer/ssl/certificate";
 constexpr std::string_view INDEXER_SSL_KEY = "/indexer/ssl/key";
 constexpr std::string_view INDEXER_SSL_USE_SSL = "/indexer/ssl/use_ssl";
 constexpr std::string_view INDEXER_SSL_VERIFY_CERTS = "/indexer/ssl/verify_certificates";
+constexpr std::string_view INDEXER_SSL_CA_MERGED = "/indexer/ssl/certificate_authorities/tmp/root-ca-merged.pem";
```

Comment:
Here we are re-introducing the original bug
https://github.com/wazuh/wazuh/pull/27250/files#diff-9891db48ea0307e404c3d5264da49bdc77acaee3052a5f37bfab090fec188743R22